nixos/modules/services/misc/flaresolverr.nix

```diff
@@ -0,0 +1,50 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+
+with lib;
```

Comment:
many people discourage the use of such a broad `with` statement these days - to be clear i am not telling you to change this, i am just telling you that there are other people who will